main.go

```diff
@@ -15,7 +15,7 @@ func main() {
 	logger := log.Default()
 
 	// * Config
-	config, err := config.Load(CONFIG_FILE)
+	config, err := config.Load(CONFIG_FILE, DUMP_MIGRATIONS, logger)
```

Comment:
👀 config gets shadowed here

Reply:
Changed the variable to `conf`. Btw, this happens in many files in the codebase, not just here, should we address this in the whole codebase? If so, we should likely enforce this through linting.